packages/@react-aria/overlays/src/usePreventScroll.ts

```diff
@@ -13,19 +13,33 @@
 import {useEffect} from 'react';
 
 /**
- * Prevents scrolling on the document body on mount, and 
+ * Prevents scrolling on the document body on mount, and
  * restores it on unmount. Also ensures that content does not
  * shift due to the scrollbars disappearing.
  */
-export function usePreventScroll(): void {
+
+interface PreventScrollOptions {
+  isOpen?: boolean
+}
+
+export function usePreventScroll(options? : PreventScrollOptions) {
```

Comment:
```suggestion
export function usePreventScroll(options? : PreventScrollOptions = {}) {
```
Adding the above would make it so you can properly attempt to deconstruct `isOpen` even when options is undefined.

Reply:
Thanks @LFDanLu! I was getting an error (How do I fix error TS1015: Parameter cannot have question mark and initializer?
). However, i stumbled upon this (https://stackoverflow.com/questions/17186566/how-do-i-fix-error-ts1015-parameter-cannot-have-question-mark-and-initializer) and it is all resolved now.

Reply:
Thanks @LFDanLu! I was getting an error (How do I fix error TS1015: Parameter cannot have question mark and initializer?
). However, i stumbled upon this (https://stackoverflow.com/questions/17186566/how-do-i-fix-error-ts1015-parameter-cannot-have-question-mark-and-initializer) and it is all resolved now.